flink/v1.13/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -318,6 +318,30 @@ public void testAlterTable() throws TableNotExistException {
     Assert.assertEquals(properties, table("tl").properties());
   }
 
+  @Test
+  public void testAlterPrimaryKeyTable() throws TableNotExistException {
```

Comment:
can we also add the coverage for expected `UnsupportedOperationException` with schema change?